packages/core/src/drivers/db/models/diligencia.js

```diff
@@ -0,0 +1,65 @@
+const { Model, DataTypes } = require('sequelize');
+const { USUARIO_TABLE } = require('./persona');
+
+const DILIGENCIA_TABLE = 'diligencia';
+
+const DiligenciaSchema = {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER,
+  },
+  personaId: {
+    allowNull: false,
+    type: DataTypes.INTEGER,
+    field: 'persona_id',
+    references: {
+      model: USUARIO_TABLE,
+      key: 'id',
+    },
+  },
+
```

Comment:
In this case, we don't need the table of USUARIOS, because it will not be a user, it will need only the table PERSONA, to fill that fields.